README.md

```diff
@@ -24,13 +24,34 @@ We support following modules right now.
 * [ChainIO](./chainio)
 * [Services](./services)
 
+## AVS use examples
+
+This SDK has three AVS use examples:
+
+* [Incredible Squaring](https://github.com/Layr-Labs/eigensdk-go/tree/v2-dev-2/examples/incredible-squaring)
+* [Incredible Dot Product](https://github.com/Layr-Labs/eigensdk-go/tree/v2-dev-2/examples/incredible-dot-product)
+* [Awesome Vault Service](https://github.com/Layr-Labs/eigensdk-go/tree/v2-dev-2/examples/awesome-vault-service)
+
+These are examples of the code needed to create an AVS from scratch. All include the contracts needed to execute them, with further instructions in their respective readme files.
+
+### Business logic in entities
+
+* Aggregator: The aggregator does not have much business logic, since the task processor implementation lies in the Indexing Task Processor implementation, which can be seen as the default one. If wanted to create your task processor, you can base it on the ITP implementation, and change what you need.
+* Challenger: The challenger business logic lies in Task response validation. To validate the response, the challenger first calculates the response with the same function as the operator and then compares it with the received response, raising a challenge if they differ.
+* Operator: The operator responds to tasks using the VaultServiceResponseCalculator struct, which has a ComputeResponse method (satisfying the ResponseCalculator interface).
```

Comment:
This part has specifics of the Awesome Vault Service too (VaultServiceResponseCalculator).